accounts/modules/role_policies/ci/policies.tf

```diff
@@ -24,6 +24,8 @@ data "aws_iam_policy_document" "ci_permissions" {
       "ecr-public:InitiateLayerUpload",
       "ecr-public:UploadLayerPart",
       "ecr-public:CompleteLayerUpload",
+      "lambda:UpdateFunctionCode",
```

Comment:
Might be worth a comment to delineate it from the ECR policies described in the comment above?